test/python/WMComponent_t/WorkflowUpdater_t/WorkflowUpdater_t.py

```diff
@@ -96,6 +96,7 @@ def getConfig(self):
         config.WorkflowUpdater.rucioUrl = "http://cms-rucio-int.cern.ch"
         config.WorkflowUpdater.rucioAuthUrl = "https://cms-rucio-auth-int.cern.ch"
         config.WorkflowUpdater.msPileupUrl = "https://cmsweb-testbed.cern.ch/ms-pileup/data/pileup"
+        config.WorkflowUpdater.sandboxDir = "/tmp"
```

Comment:
We don't need this component attribute (as mentioned in the WorkflowUpdaterPoller class).

Reply:
ok, removed